db/migrate/20230313090203_create_buses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBuses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :buses do |t|
+      t.string :name,         null: false, default: ""
+      t.string :type,         null: false, default: ""
+      t.string :brand,         null: false, default: ""
+      t.integer :capacity,         null: false, default: 0
+
+      t.timestamps
```

Comment:
do we really need default value as ""?
what do you think?

Reply:
removed , thank you